readme.md

```diff
@@ -14,31 +14,15 @@ A [clamping function](https://en.wikipedia.org/wiki/Clamping_(graphics)) constra
 
 The primary motivation for this function is to bring parity with the [CSS function][css-clamp] of the same name. It will be useful when using it in conjunction with the [CSS Painting API](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Painting_API).
```

Comment:
I'm skeptical of the use of CSS behavior here, which is probably not available in ES from a programming language level.

I invite @hax experts to come and give further guidance.

Reply:
It appears that the motivation behind this proposal is to provide a similar functionality to the CSS function `clamp` in JavaScript. However: 

1. The JavaScript version of `clamp` does not support CSS units. And it seems JS will never support them because of the death of https://github.com/tc39/proposal-extended-numeric-literals .
2. The parameter order in the draft JavaScript version is `(value, min, max)`, whereas the CSS version is `(min, value, max)`, which is quite confusing.
3. The CSS version's prioritization of the minimum value is based on accessibility needs in CSS practices. However, this behavior might not be easily mapped to general-purpose programming languages, as pointed out in this pull request. It seems to be quite inconsistent with the behavior of other programming languages.